src/MizarBase/include/MizarBase/Time.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_BASE_TIME_H_
+#define MIZAR_BASE_TIME_H_
+
+#include <utility>
+
+#include "OrbitBase/Typedef.h"
+#include "stdint.h"
+
+namespace orbit_mizar_base {
+
+struct RelativeTimestampTag : orbit_base::PlusTag {};
```

Comment:
The naming `relative` confusing me a bit, aka I am not sure what you actually mean by that. This seems like something I would call `duration`. If you want to stick with `relative`, an explaining comment would be great.

Reply:
I'll add the comment "Represents the time passed since the capture start"